packages/lesswrong/server/resolvers/jargonResolvers/jargonTermMutations.ts

```diff
@@ -0,0 +1,256 @@
+import { jargonBotClaudeKey } from '@/lib/instanceSettings';
+import { defineMutation } from '../../utils/serverGraphqlUtil';
+import { getAnthropicPromptCachingClientOrThrow } from '../../languageModels/anthropicClient';
+import { exampleJargonGlossary2, exampleJargonPost2 } from './exampleJargonPost';
+import { userIsAdmin } from '@/lib/vulcan-users';
+import { PromptCachingBetaMessageParam } from '@anthropic-ai/sdk/resources/beta/prompt-caching/messages';
+import JargonTerms from '@/lib/collections/jargonTerms/collection';
+import { createMutator } from '../../vulcan-lib';
+import { initialGlossaryPrompt, formatPrompt, glossarySystemPrompt, mathFormatPrompt } from './jargonPrompts';
+import { fetchFragmentSingle } from '@/server/fetchFragment';
+import { htmlToMarkdown } from '@/server/editor/conversionUtils';
+import { exampleMathGlossary } from './exampleMathOutput';
+import { readFile } from 'fs/promises';
+import { userCanCreateAndEditJargonTerms } from '@/lib/betas';
+
+const claudeKey = jargonBotClaudeKey.get()
+
+async function queryClaudeJailbreak(prompt: PromptCachingBetaMessageParam[], maxTokens: number, systemPrompt: string) {
+const client = getAnthropicPromptCachingClientOrThrow(claudeKey)
+return await client.messages.create({
+    system: systemPrompt,
+    model: "claude-3-5-sonnet-20240620",
+    max_tokens: maxTokens,
+    messages: prompt
+})
+}
+
+export const queryClaudeForTerms = async (markdown: string) => {
+  console.log(`I'm pinging Claude for terms!`)
+  const termsResponse = await queryClaudeJailbreak([
```

Comment:
This really feels like it should be doing tool-use, unless there's a compelling reason to not do that.